drivers/mfd/Kconfig.aw9523b

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+config MFD_AW9523B
+	bool "Awinic AW9523B GPIO/LED controller device driver"
+	default y
+	depends on DT_HAS_AWINIC_AW9523B_ENABLED
+	select I2C
+	help
+	  Enable the Awinic AW9523B GPIO/LED controller device driver
+
+config MFD_AW9523B_INIT_PRIORITY
+	int "nPM1300 MFD initialization priority"
```

Comment:
wrong comment

Reply:
Thank you for pointing that out. This is embarrassing.
Fixed it.